router/core/graph_server.go

```diff
@@ -675,26 +673,21 @@ func (s *graphServer) buildGraphMux(ctx context.Context,
 
 	httpRouter := chi.NewRouter()
 
-	baseOtelAttributes := append([]attribute.KeyValue{otel.WgRouterConfigVersion.String(routerConfigVersion)}, s.baseOtelAttributes...)
-
-	if featureFlagName != "" {
-		baseOtelAttributes = append(baseOtelAttributes, otel.WgFeatureFlag.String(featureFlagName))
-	}
+	exprManager := expr.CreateNewExprManager()
 
 	metricsEnabled := s.metricConfig.IsEnabled()
 
 	// we only enable the attribute mapper if we are not using the default cloud exporter
-	enableAttributeMapper := !(s.metricConfig.IsUsingCloudExporter || rmetric.IsDefaultCloudExporterConfigured(s.metricConfig.OpenTelemetry.Exporters))
-
-	exprManager := expr.CreateNewExprManager()
+	baseMuxAttributes := getBaseMuxAttributes(routerConfigVersion, s.baseOtelAttributes, featureFlagName)
```

Comment:
Do we need to do this twice? Can't we collect the attributes only once before (for NewRuntimeMetrics) and add here only the feature flag attribute? I find it hard to read when we introduce smaller functions mapping functions that essentially nothing doing.